assets/buttonsActions.js

```diff
@@ -0,0 +1,101 @@
+// LANGUAGE WRAPPER //
+
+const wrapper = document.querySelector('#languageWrapper');
+wrapper.addEventListener('click', () => {
+  wrapper.classList.toggle('viewMenu');
+});
+
+document.addEventListener('click', (e) => {
+  if (e.target.closest('#languageWrapper')) return;
+  wrapper.classList.remove('viewMenu');
+});
+
+// LANGUAGE WRAPPER //
+
+// MENU BUTTONS ACTION
+
+const body = document.querySelector('body');
+const menu = document.querySelector('#menuBar');
+const menuButtons = document.querySelectorAll('.menuAction');
+const hamburguer = document.querySelector('#menuBtn');
+
+menuButtons.forEach((index) => {
```

Comment:
- [ ] Your hamburger is working great but there is a small issue with it, when menu is displayed and screen resizes the mobile menu disappears but the icons are not toggled. When we resize back to the mobile resolution the `X` icon is displaying in place of `hamburger icon`  and  vice versa. 


https://user-images.githubusercontent.com/44798044/226086256-b2ba7fb3-a44c-4019-90f9-e77b4a987a93.mov

Reply:
@shella12 Got it! I've already corrected that too, thanks! 

https://user-images.githubusercontent.com/92229666/226092332-30e0774f-6b4a-4eee-a5d1-10085f74f254.mp4